src/resmoke/resmoke_proxy.rs

```diff
@@ -41,7 +41,7 @@ pub struct ResmokeProxy {
     /// Primary command to invoke resmoke (usually `python`).
     resmoke_cmd: String,
     /// Script to invoke resmoke.
-    resmoke_script: String,
+    resmoke_script: Vec<String>,
```

Comment:
If this script is composed of multiple arguments, then it needs to be interpreted as a vector not a string or the shell will not use it correctly.